samples/subsys/display/lvgl/src/main.c

```diff
@@ -54,7 +101,7 @@ void main(void)
 			lv_label_set_text(count_label, count_str);
 		}
 		lv_task_handler();
-		k_sleep(K_MSEC(10));
 		++count;
+		k_sleep(K_MSEC(10));
```

Comment:
Useless diff / unrelated change, it looks like.

Reply:
Not really, pressing the button resets the value immediately now, otherwise it would jump to 1 after a second

Reply:
Sorry I was unclear. If it's not useless diff, it's an unrelated change and the commit is doing 2 things.

Reply:
I would argument that it belongs with the commit `samples: subsys: display: lvgl: Add button to reset counter` as this introduces the new functionality.
Creating a new commit feels like a fixup.